app/controllers/ReportingFrequencyPageController.scala

```diff
@@ -70,13 +70,16 @@ class ReportingFrequencyPageController @Inject()(
             }
           }
 
+          val isCeased = user.incomeSources.businesses.forall(_.isCeased)
```

Comment:
Could we move this to IncomeSourceDetailsResponseModel please? It helps to reuse in other places. Something like:

`def areAllBusinessesCeased: Boolean = businesses.forall(_.isCeased) `